common/crypto/openssl/ec.c

```diff
@@ -560,7 +836,7 @@ bool oe_ec_valid_raw_private_key(
     if (bn != NULL)
         BN_clear_free(bn);
     if (group != NULL)
-        EC_GROUP_clear_free(group);
+        EC_GROUP_free(group);
```

Comment:
Can you check if `EC_GROUP_free` be sufficient?
For 1.1.1 (https://www.openssl.org/docs/man1.1.1/man3/EC_GROUP_clear_free.html):
```
EC_GROUP_free() frees the memory associated with the EC_GROUP. If group is NULL nothing is done.

EC_GROUP_clear_free() destroys any sensitive data held within the EC_GROUP and then frees its memory. If group is NULL nothing is done.
```

Not sure about OpenSSL 3.

Reply:
`EC_GROUP_clear_free()` is deprecated in 3. Its functionality is now part of EC_GROUP_free()

Reply:
I get that. But now this change applies to both OpenSSL 1 and 3. My question was
For OpenSSL 1, would using `EC_GROUP_free` be sufficient (without introdcuing memory leak)?
Also, please check if it's sufficient on OpenSSL 3 as well.

Reply:
I see what you mean. I added a OpenSSL 3 version like the rest.